hw/ip/kmac/data/kmac.hjson

```diff
@@ -520,9 +519,16 @@
               Only after that, the module can start processing commands.
               Depending on !!CFG_SHADOWED, the internal PRNG is then used for (re-)masking inputs (prefix, key, message) and intermediate results of the Keccak core.
 
-              Depending on !!ENTROPY_PERIOD, the module will periodically reseed the internal PRNG with fresh entropy from EDN.
-              Using !!CMD.entropy_req software can manually initiate the reseeding.
+              The module will then automatically reseed the internal PRNG with fresh entropy from EDN every !!ENTROPY_REFRESH_THRESHOLD_SHADOWED messages in KMAC mode.
+              More precisely, automatic reseed operations are always triggered directly after the key absorption phase.
+              How close the module is automatically trigger a reseed operation can be inferred from !!ENTROPY_REFRESH_HASH_CNT.
```

Comment:
I think you probably want "How close the module is to automatically triggering ..." ?